extensions/blocks/map/add-point/style.scss

```diff
@@ -41,6 +41,6 @@
 	box-shadow: none;
 	float: right;
 	path {
-		color: #aaa;
+		color: $dark-gray-150;
```

Comment:
Before
<img width="273" alt="image" src="https://user-images.githubusercontent.com/87168/56816486-4ff88100-684c-11e9-94bc-3b519d8d4f37.png">

After

<img width="274" alt="Screenshot 2019-04-26 at 17 06 28" src="https://user-images.githubusercontent.com/87168/56816471-4838dc80-684c-11e9-81b3-4ea71341e71d.png">

New color is _"lightest gray that can be used for AA non-text contrast."_